### PR TITLE
Updated .travis.yml (sudo: false, php 7.0, php 7.1, cache)

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 
+sudo: false
+
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```
Comment (GrahamCampbell) [inline .travis.yml]: 7.1 can be added there too

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   - hhvm
 
 before_script:
-  - composer install --prefer-source --no-interaction
```
Comment (GrahamCampbell) [inline .travis.yml]: these flags are fine to stay

Comment (GrahamCampbell) [inline .travis.yml]: it's selfish to use all travis' api rate limit to download deps ove dist

Comment (lyrixx) [inline .travis.yml]: Oh, I though the API rate were removed (see: https://github.com/composer/composer/issues/4884#issuecomment-195229989)

File: .travis.yml (hunk 1 above)
Comment (lyrixx) [inline .travis.yml]: fixed thanks.

File: .travis.yml (hunk 2 above)
Comment (GrahamCampbell) [inline .travis.yml]: Oh, yeh, forgot about that! This looks good then! 👍

**Comment by stof** [comment]
you should also enable the cache for ``$HOME/.composer/cache/files`` to cache the composer downloads between builds (Travis does not do it automatically yet, even though there is a discussion in an issue about supporting the composer cache folder automatically, as they do for bundler)

**Comment by lyrixx** [comment]
@stof Thanks. Done ;)

**Pull request merged**